### PR TITLE
Reduce node update frequency

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/enhancements/issues/589

```release-note
Reduce default NodeStatusReportFrequency to 5 minutes. With this change, periodic node status updates will be send every 5m if node status doesn't change (otherwise they are still send with 10s).

Bump NodeProblemDetector version to v0.8.0 to reduce forced NodeStatus updates frequency to 5 minutes.
```

### CODE REVIEW

**Comment by smourapina** [comment]
@wojtek-t @gmarek @Random-Liu Bug triage for 1.17 here with a gentle reminder that code freeze for this release is on November 18. Is this PR still intended for 1.17?

**Comment by wojtek-t** [comment]
Yes - it is. And it will be merged by then.

File: cluster/gce/gci/configure.sh [hunk 1]
```diff
@@ -26,8 +26,8 @@ set -o pipefail
 ### Hardcoded constants
 DEFAULT_CNI_VERSION="v0.7.5"
 DEFAULT_CNI_SHA1="52e9d2de8a5f927307d9397308735658ee44ab8d"
-DEFAULT_NPD_VERSION="v0.7.1"
-DEFAULT_NPD_SHA1="a9cae965973d586bf5206ad4fe5aae07e6bfd154"
+DEFAULT_NPD_VERSION="v0.8.0"
+DEFAULT_NPD_SHA1="276335fa25a703615cc2f2cdc51ba693fac4bdd70baa63f9cbf228291defd776"
```
Comment (wojtek-t) [inline cluster/gce/gci/configure.sh]: @Random-Liu - it seems the sha is incorrect. Where I can get sha1 from?

Comment (wojtek-t) [reply cluster/gce/gci/configure.sh]: never mind

**Comment by wojtek-t** [comment]
@derekwaynecarr @Random-Liu - PTAL

**Comment by derekwaynecarr** [comment]
bumps time to previously discussed interval during kep discussion.

/lgtm
/approve

**Pull request merged**